.github/workflows/maven.yml

```diff
@@ -0,0 +1,13 @@
+name: Java Sandbox Agent
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 1.7
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.7
```

Comment:
Does 1.7 still exist?

Reply:
Has to, since we still support the JAVA7 executor.